### DISCUSSION TITLE
ELRS for Pixhawk 2.4.8 with RP3 ExpressLRS 2.4ghz Nano Receiver and TX12 Mark II Radio Controller

### DISCUSSION THREAD

**Discussion by SANTHOSH-MAMIDISETTI** [ExpressLRS/ExpressLRS#2456] [Q&A]
I am very new to this field , this is my first build with Pixhawk 2.4.8 and Mission planner. I am planning to buy TX12 Mark II Radio Controller (ELRS version) , please help me choose a receiver, I am unable to figure out what is needed for my quadcopter and also what is the difference between RP1 ,RP2,RP3 ELRS Nano Receivers and ER4 ER6 ER8 2.4GHz ELRS PWM receivers? please help me  and also do I only have to buy the receiver or any other extra hardware with it ?

And also If I buy [RP3 ExpressLRS 2.4ghz Nano Receiver](https://www.radiomasterrc.com/products/rp3-expresslrs-2-4ghz-nano-receiver?variant=46464183075047) for my [TX12 Mark II Radio Controller](https://www.radiomasterrc.com/products/tx12-mark-ii-radio-controller?variant=4412845) (MODE 2, FCC) for my Pixhawk 2.4.8 and Mission planner(1.3.80) and Copter 4.4.1 official firmware. does ELRS work or would I have to buy any extra hardware to interface them to make it work. ( I am not sure , but something like a PPM converter ?). I am very new to these and confused ,

**Comment by DanT2000** [comment]
Уже сейчас понимаю что прошел год и ты разобрался что и как, но для тех кто будет искать напишу следующую инфу, по умолчанию прошивка PX4 для работы с протоколом CRSF(ELRS туда же) не подходит. Так же и многие прошивки именно по умолчанию не поддерживают, но у Pixhawk есть конфигуратор https://docs.px4.io/main/en/telemetry/crsf_telemetry.html вот инструкция что и к чему. Но скажу что путь это не легкий, по причине того что инфы не так уж и много а еще и pixhawk обычно не оригинал а реплика из Китая , не обманывайтесь 9 из 10  pixhawk 2.4.8 которые я видел китайские. Поэтому могут возникнуть сложности хотя и они решаются если у вас есть время. Поэтому если вы новичок. То возьмите переходник за 15$ и дальше когда уже будет аппарат способный летать , и вы получите стимул сделать без него , вот тогда переходите дальше к костюмным прошивкам, там все круто , но не без проблем.
